src/transport/abstract-transport.js

```diff
@@ -20,6 +20,7 @@ function AbstractTransport(address, timeout, messageHandler) {
   this.awaitingPump = false;
   this.connected = false;
   this.backoffMilliseconds = 1;
+  this.nextReconnectListenerToken = 1;
```

Comment:
_Assuming_ all of this code is only ever used in this one way and nothing ever changes over time, then I believe you are correct that `ethrpc.subscribeNewHeads` will only be called once per instantiation of `BlockStream` and there will likely only be one token ever used.